app/templates/_proposition_header.html

```diff
@@ -4,12 +4,16 @@
     <nav id="proposition-menu">
       <a href="/" id="proposition-name">Digital Marketplace</a>
       <ul id="proposition-links">
-        {% if current_user.email_address %}
+        {% if current_user.is_authenticated() %}
+          {% if current_user.role == 'buyer' %}
+            <li><a href="{{ url_for('buyers.buyer_dashboard') }}">View your account</a></li>
```

Comment:
Looking good. Could maybe only make the url conditional, rather than the whole `li`, e.g.
```
 <li>
<a href="{{ url_for('buyers.buyer_dashboard') if  current_user.role == 'buyer' else '/suppliers' }}">
```